src/core/trtserver.cc

```diff
@@ -923,6 +923,32 @@ TRTSERVER_ServerModelStatus(
   return nullptr;  // success
 }
 
+TRTSERVER_Error*
+TRTSERVER_LoadModel(TRTSERVER_Server* server, const char* model_name)
+{
+  ni::InferenceServer* lserver = reinterpret_cast<ni::InferenceServer*>(server);
+
+  ni::ServerStatTimerScoped timer(
+      lserver->StatusManager(), ni::ServerStatTimerScoped::Kind::CONTROL);
```

Comment:
It seems that we should record both success and failed control requests. This can be done as a separate change.